services/expiration/expiration-notification.server.ts

```diff
@@ -189,8 +189,8 @@ const getExpiringDnsRecords = () => {
 };
 // function to add jobs
 async function addExpirationNotifications(type: string) {
-  let jobName = `${expirationNotificationQueueName}-${type}-expiry`;
-  return expirationNotificationQueue.add(jobName, {
+  let jobId = `${expirationNotificationQueueName}-${type}-expiry`;
+  return expirationNotificationQueue.add(expirationNotificationQueueName, jobId, {
```

Comment:
I think the `jobId` has to be part of `data` passed in here.  Should you not wrap this in `{...}`:

```ts
return expirationNotificationQueue.add(expirationNotificationQueueName, { jobId }, {
```

Confirm that I'm right, but I think you have to do this.

Reply:
You need to fix this, which is why I'm not approving this.

Reply:
I don't see duplicate emails, but I could add this.